apps/backoffice/src/lib/be/utils/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { BackOfficeUser } from "../../../../types/next-auth";
+
+export const isBackofficeUserAdmin = (user: BackOfficeUser) =>
+  user.institution.role === "ADMIN" ? true : false;
```

Comment:
Nit pick: can be written just as

''' lang=ts
export const isBackofficeUserAdmin = (user: BackOfficeUser) =>
  user.institution.role === "ADMIN";
'''